lib/msal-angular/tests/MSALSpec.ts

```diff
@@ -51,7 +51,6 @@ describe('Msal Angular Pubic API tests', function () {
                     useValue: {
                         popUp: false,
                         consentScopes: ["user.read", "mail.send"],
-                        unprotectedResources: ["https://google.com"],
```

Comment:
Removing this line before the fix reproduced the bug, so keeping it removed to prove it is fixed.